test/e2e/es_cluster_logging.go

```diff
@@ -201,7 +205,7 @@ func ClusterLevelLoggingWithElasticsearch(f *Framework) {
 	// Form a unique name to taint log lines to be collected.
 	// Replace '-' characters with '_' to prevent the analyzer from breaking apart names.
 	taintName := strings.Replace(ns+name, "-", "_", -1)
-
+	Logf("Generated taintName %v for query exact match query", taintName)
```

Comment:
minor nit: the phrase "Tainting log lines with %v" might be a little bit more descriptive of what this is.